ecosystem/typescript/sdk/examples/javascript/multisig_transactiond.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-console */
```

Comment:
I'm wondering, what's the reason for naming the file multisig_transaciton**d**?

Reply:
typo. will fix :)